StreamChat.xcodeproj/project.pbxproj

```diff
@@ -13985,10 +13984,10 @@
 			buildSettings = {
 				ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES = YES;
 				CLANG_CXX_LANGUAGE_STANDARD = "gnu++17";
-				CODE_SIGN_STYLE = Manual;
+				CODE_SIGN_IDENTITY = "Apple Development";
+				CODE_SIGN_STYLE = Automatic;
```

Comment:
Why are we changing this?

Reply:
The cert got expired, I updated it and now it fails for some reason to sign the Test app. So I set it to automatic signing, seems to be working fine now 🤔